### PR TITLE
SLES 15sp1 repos

### PR DESCRIPTION
**Pull request by cbosdo**
Add the 15SP1 repos for HEAD and TEST

### CODE REVIEW

File: salt/mirror/minima.yaml [hunk 1]
```diff
@@ -35,6 +35,26 @@ scc:
...
+    - SLE-Product-SLES15-SP1-Pool
+    - SLE-Product-SLES15-SP1-Updates
+    - SLE-Product-WE15-SP1-Pool
+    - SLE-Product-WE15-SP1-Updates
+    # SLE 15-SP1 Basic Modules
+    - SLE-Module-Basesystem15-SP1-Pool
+    - SLE-Module-Basesystem15-SP1-Updates
+    - SLE-Module-Server-Applications15-SP1-Pool
+    - SLE-Module-Server-Applications15-SP1-Updates
+    - SLE-Module-Desktop-Applications15-SP1-Pool
```
Comment (juliogonzalez) [inline salt/mirror/minima.yaml]: Interesting. I wonder why we are adding Desktop Applications (I see it was added before, for SLE15 GA)

Server requires: Base, Server Applications, Web Scripting Python 2
Proxy requires: Base, Server Applications

Shall we try without Desktop? It seems we are not using it below.

File: salt/repos/default.sls [hunk 2]
```diff
@@ -278,6 +258,43 @@ tools_additional_repo:
...
+    - name: /etc/zypp/repos.d/SLE-15-SP1-x86_64-Pool.repo
+    - source: salt://repos/repos.d/SLE-15-SP1-x86_64-Pool.repo
+    - template: jinja
+
+os_update_repo:
+  file.managed:
+    - name: /etc/zypp/repos.d/SLE-15-SP1-x86_64-Update.repo
+    - source: salt://repos/repos.d/SLE-15-SP1-x86_64-Update.repo
+    - template: jinja
+{% endif %} {# '15.1' == grains['osrelease'] #}
```
Comment (juliogonzalez) [inline salt/repos/default.sls]: Maybe we should ad the unreleased updates, for the future.

File: salt/mirror/minima.yaml (hunk 1 above)
Comment (cbosdo) [reply salt/mirror/minima.yaml]: I'm sure it would work without them

File: salt/repos/default.sls (hunk 2 above)
Comment (cbosdo) [reply salt/repos/default.sls]: how? there is no repo for them yet?

Comment (juliogonzalez) [reply salt/repos/default.sls]: I'd copy and then adapt `SLE-15-x86_64-Test-Update.repo`. I don't think the new repository for SP1 is going to have fancy changes.

But this is just a suggestion, not something required to merge the PR :-)

File: salt/mirror/minima.yaml (hunk 1 above)
Comment (juliogonzalez) [reply salt/mirror/minima.yaml]: Le'ts do the following: open an issue to see if we can remove the Desktop Applications module from SLE15 and SLE15-SP1, and let's merge this as it is.

File: salt/repos/default.sls (hunk 2 above)
Comment (mcalmer) [reply salt/repos/default.sls]: Desktop should go away. If there is something in Desktop what we need, we should try to get it moved or we copy it in our own repo.

Comment (cbosdo) [reply salt/repos/default.sls]: the problem is that http://download.suse.de/ibs/SUSE:/Maintenance:/Test:/SLE-Module-Basesystem:/ doesn't have a 15SP1 folder yet... we'll need to add it later

Comment (mcalmer) [reply salt/repos/default.sls]: I think we should use the official betas (from SLES) for testing. No intermediate builds.

**Pull request merged**